modules/packages/CopyAggregation.chpl

```diff
@@ -112,10 +112,22 @@ module CopyAggregation {
     type elemType;
```

Comment:
It looks like the elemType field is not documented either.  Totally fine for that to not be your problem, but I figured I should mention it.

Reply:
Yeah, I definitely see that.  Still would be nice to be explicit, but I'm okay with that not happening in this PR